testing/fuchsia/run_tests.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024, the Flutter project authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found
+# in the LICENSE file.
+
+import argparse
+import os
+import sys
+from typing import List
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(__file__), '../../../fuchsia/test_scripts/test/'
+    )
+)
+
+import run_test
+from run_executable_test import ExecutableTestRunner
+from test_runner import TestRunner
+
+
+# TODO(https://github.com/flutter/flutter/issues/140179): Execute all the tests
+# in
+# https://github.com/flutter/engine/blob/main/testing/fuchsia/test_suites.yaml
+def _get_test_runner(
+    runner_args: argparse.Namespace, test_args: List[str]
+) -> TestRunner:
+  return ExecutableTestRunner(
+      runner_args.out_dir, [],
+      'fuchsia-pkg://fuchsia.com/dart_runner_tests#meta/dart_runner_tests.cm',
+      runner_args.target_id, 'codecoverage', '/tmp/log',
+      ['out/fuchsia_debug_x64/dart_runner_tests.far'], None
+  )
+
+
+# TODO(https://github.com/flutter/flutter/issues/140179): Respect build
+# configurations.
+if __name__ == '__main__':
```

Comment:
Can we add this to https://github.com/flutter/engine/blob/main/ci/builders/linux_fuchsia.json to start running the test in CI?

Reply:
Yes, it's the reason of the TODO. I am still investigating the way of getting build configurations to avoid the hardcoded path. I can add it to the debug x64 combination first.

Reply:
Added, e.g. https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8759257676482292513/+/u/test:_x64_emulator_based_tests/stdout

But there are two issues,
1) The error did not trigger a failure.
2) The kvm is not available.